In [1]:
# Importando os pandas
import pandas as pd

In [2]:
df = pd.read_csv('clima.csv') # Criando um data frame 

### Usando o campo Umidade9,Temperatura9, VelocidadeVento9

In [3]:
colunas_pedidas = ['Umidade9', 'Temperatura9', 'VelocidadeVento9']
df[colunas_pedidas].describe()

,Umidade9,Temperatura9,VelocidadeVento9
count,142806.000000,143693.000000,143693.000000
mean,68.880831,16.990631,14.043426
std,19.029164,6.488753,8.915375
min,0.000000,-7.200000,0.000000
25%,57.000000,12.300000,7.000000
50%,70.000000,16.700000,13.000000
75%,83.000000,21.600000,19.000000
max,100.000000,40.200000,130.000000


### 1 - Quantos dados existem ?

In [4]:
df[colunas_pedidas].count()

Umidade9            142806
Temperatura9        143693
VelocidadeVento9    143693
dtype: int64

### Explicação:
Os valores acima indica o total de dados. Notei que na Umidade9, alguns valores foram retirados, que são os valores nulos, que não foram contabilizados.

### 2 - Qual é a média dos valores ?

In [5]:
df[colunas_pedidas].mean()

Umidade9            68.880831
Temperatura9        16.990631
VelocidadeVento9    14.043426
dtype: float64

### Explicação:

Com referencia da média dos dados acima, a Umidade9 esta ideal e confortável. A média das temperatura9, está motrando que a temperatura tende a ser fria. E a velociade do vento está fraca, com velocidades em média de 14km/h.

### 3 - Qual o valor mínimo dos valores ?

O valor mínimo dos valores foi resgistrado em Temperatura9 com um registro de -7.200000

### 4 - Qual o valor máximo dos valores ?

O valor máximo registrado foi VelocidadeVento9, com uma velocidade do vento de 130.000000.

### 5 - Qual é a mediana ?

In [6]:
df[colunas_pedidas].median()

Umidade9            70.0
Temperatura9        16.7
VelocidadeVento9    13.0
dtype: float64

### 6 - 25% e 75% -Como os dados estão distribuídos ?

A maior parte dos dados está distribuída entre 25%	57.000000, 12.300000, 7.000000 e 75% 83.000000, 21.600000, 19.000000, pois esse intervalo representa o primeiro e o terceiro quartil.

### 7 - O quanto os dados variam ?

A umidade9 apresenta uma variação moderada para alta, porque o desvio padrão de 19 é relativamente considerável em relação à média de 68.

A temperatura9 apresenta uma variação moderada, pois o desvio padrão é menor em relação à média.

A velocidade do vento apresenta uma variação relativamente alta, porque o desvio padrão está bem próximo da média.
